taskset.go

```diff
@@ -151,7 +151,7 @@ func (ts *TaskSet) MustLoadTask(task Task) Task {
 
 // LoadTask adds a task to the TaskSet, but only if it has a new uuid or no uuid.
 // Return annotated task.
-func (ts *TaskSet) LoadTask(task Task) (*Task, error) {
+func (ts *TaskSet) LoadTask(task *Task) (*Task, error) {
```

Comment:
huh? we're now allowing to pass in a pointer, which can be nil, but then we dereference it without checking if it's nil?
that's recipe for a panic.

Reply:
Good catch, thanks @Dieterbe 

I've reverted this behavior in all the changes now. No reason to do the passing by ref just for `LoadTask`, so to keep consistency I reverted it to return an empty task on error.